README.Rmd

```diff
@@ -67,27 +67,30 @@ Because our office (mostly) cannot observe individual condo unit characteristics
 library(dplyr)
 library(tidyr)
 library(yaml)
-params <- read_yaml("params.yaml")
+library(httr)
+
+condo_params <- read_yaml("params.yaml")
+condo_preds <- condo_params$model$predictor$all
+
+res_params_text <- GET(
+  "https://raw.githubusercontent.com/ccao-data/model-res-avm/master/params.yaml"
+) %>%
+  content(as = "text")
+res_params <- read_yaml(text = res_params_text)
+res_preds <- res_params$model$predictor$all
+
+condo_unique_preds <- setdiff(condo_preds, res_preds)
+
 ccao::vars_dict %>%
-  filter(
-    var_is_predictor,
-    !var_name_model %in% c("meta_sale_price")
-  ) %>%
```

Comment:
It seems to me like this filter is unnecessary, since the subsequent join should remove any columns that don't match these conditions. I removed it and it doesn't seem to cause problems.